### ISSUE TITLE
[PutRequestResponseSchemeArm] - false positive

### ISSUE THREAD

**Issue by ms-zhenhua** [Azure/azure-openapi-validator#545]
**Describe the bug**
The following request and response use the same model, but lint-diff still reported the error.
```
"put": {
        "tags": [
          "Workspaces"
        ],
        "summary": "Creates or updates a workspace with the specified parameters.",
        "operationId": "Workspaces_CreateOrUpdate",
        "consumes": [
          "application/json"
        ],
        "produces": [
          "application/json"
        ],
        "parameters": [
          {
            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
          },
          {
            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
          },
          {
            "$ref": "#/parameters/WorkspaceNameParameter"
          },
          {
            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
          },
          {
            "in": "body",
            "name": "body",
            "required": true,
            "schema": {
              "$ref": "#/definitions/Workspace"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "The request was successful; the request was well-formed and received properly.",
            "schema": {
              "$ref": "#/definitions/Workspace"
            }
          },
```

**To Reproduce**
https://github.com/Azure/azure-rest-api-specs/pull/24333/checks?check_run_id=14095108357

**Expected behavior**
No error reported

**Screenshots**
![image](https://github.com/Azure/azure-openapi-validator/assets/76987228/19de2531-11f3-4240-80fe-8ffbe8a754b6)

**Comment by atastrophic** [comment]
same issue here, were you able to figure this issue out? @ms-zhenhua

**Comment by rkmanda** [comment]
The linter rule is fixed in the staging pipeline. We will be rolling this out to production soon

**Comment by rkmanda** [comment]
The fix is rolled out
